### PR TITLE
chapter5_part5:/050_Search/20_Query_string.asciidoc

### PR DESCRIPTION
**Pull request by node**
重提 PR #286

### CODE REVIEW

**Comment by medcl** [comment]
c/github/book.xml:4269: element xref: validity error : IDREF attribute linkend references an unknown ID "all_field"

File: 050_Search/20_Query_string.asciidoc [hunk 1]
```diff
@@ -1,28 +1,21 @@
 [[search-lite]]
-=== Search _Lite_
+=== _轻量_ 搜索
 
-There are two forms of the `search` API: a ``lite'' _query-string_ version
-that expects all its((("searching", "query string searches")))((("query strings", "searching with"))) parameters to be passed in the query string, and the full
-_request body_ version that expects a JSON request body and uses a
-rich search language called the query DSL.
+有两种形式的‘搜索’ API：一种“轻量的” _查询字符串_ 版本，要求在查询字符串中传递所有的((("searching", "query string searches")))((("query stri    ngs", "searching with")))参数，另一种更完整的_请求体_版本，要求使用JSON格式，及更丰富的搜索语言，叫做查询表达式。
```
Comment (candythinking) [inline 050_Search/20_Query_string.asciidoc]: 一种“轻量的” _查询字符串_ 版本 =》 一种是“轻量”的 _查询字符串_ 版本
另一种更完整的_请求体_版本 =》 另一种是更完整的_请求体_版本
要求使用JSON格式，及更丰富的搜索语言，叫做查询表达式。=》 要求使用 JSON 格式的请求体，及更丰富的搜索语言，叫做查询的DSL。

Comment (node) [reply 050_Search/20_Query_string.asciidoc]: 查询表达式 是复用全书的术语约定，所以我还是建议与其他章节保持一致，其他已改。

**Comment by medcl** [comment]
貌似我本地还是编译不过

File: 050_Search/20_Query_string.asciidoc [hunk 2]
```diff
@@ -1,28 +1,21 @@
 [[search-lite]]
-=== Search _Lite_
+=== _轻量_ 搜索
 
-There are two forms of the `search` API: a ``lite'' _query-string_ version
-that expects all its((("searching", "query string searches")))((("query strings", "searching with"))) parameters to be passed in the query string, and the full
-_request body_ version that expects a JSON request body and uses a
-rich search language called the query DSL.
+有两种形式的‘搜索’ API：一种是“轻量的” _查询字符串_ 版本，要求在查询字符串中传递所有的((("searching", "query string searches")))((("query stri    ngs", "searching with")))参数，另一种是更完整的 _请求体_ 版本，要求使用 JSON 格式，及更丰富的搜索语言，叫做查询表达式。
```
Comment (AlixMu) [inline 050_Search/20_Query_string.asciidoc]: the full request body version that expects a JSON request body and uses a rich search language called the query DSL.
这里可不可以这样翻译：
另一种是更完整的 _请求体_ 版本，要求使用 JSON 格式和更丰富的查询表达式作为搜索语言。

Comment (node) [reply 050_Search/20_Query_string.asciidoc]: 已更新。

**Comment by node** [comment]
格式的问题是引号导致的，之前没有预览，已修复。

**Comment by medcl** [comment]
好像还是不行哦


a2x: executing: "xmllint" --nonet --noout --valid "/Users/medcl/Documents/RootSync/github/book.xml"

/Users/medcl/Documents/RootSync/github/book.xml:4117: element section: validity error : ID _search_lite already defined
<section ia2x: ERROR: "xmllint" --nonet --noout --valid "/Users/medcl/Documents/RootSync/github/book.xml" returned non-zero exit status 4
d="_search_lite">
                          ^
/Users/medcl/Documents/RootSync/github/book.xml:4193: element literal: validity error : Element emphasis is not declared in literal list of possible children
<literal><emphasis>all</emphasis></literal> 字段包含 <literal>aggregations</
                                           ^
/Users/medcl/Documents/RootSync/github/book.xml:1113: element xref: validity error : IDREF attribute linkend references an unknown ID "search-lite"
/Users/medcl/Documents/RootSync/github/book.xml:5339: element link: validity error : IDREF attribute linkend references an unknown ID "search-lite"
/Users/medcl/Documents/RootSync/github/book.xml:5666: element xref: validity error : IDREF attribute linkend references an unknown ID "search-lite"
/Users/medcl/Documents/RootSync/github/book.xml:7689: element xref: validity error : IDREF attribute linkend references an unknown ID "search-lite"

**Comment by medcl** [comment]
element literal: validity error : Element emphasis is not declared in literal list of possible children
<literal><emphasis>all</emphasis></literal> 字段包含 <literal>aggregations</
                                           ^
/Users/medcl/Documents/RootSync/github/book.xml:1113: element xref: validity error : IDREF attribute linkend references an unknown ID "search-lite"
/Users/medcl/Documents/RootSync/github/book.xml:5339: element link: validity error : IDREF attribute linkend references an unknown ID "search-lite"
/Users/medcl/Documents/RootSync/github/book.xml:5666: element xref: validity error : IDREF attribute linkend references an unknown ID "search-lite"
/Users/medcl/Documents/RootSync/github/book.xml:7689: element xref: validity error : IDREF attribute linkend references an unknown ID "search-lite"

**Pull request merged**